File: Diplom/Diplom/Models/NotificationModels/Notification.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Diplom.Models.NotificationModels
{
    public enum NotificationType
    {

    }

    public class Notification
    {
        public int NotificationId { get; set; }
        public int UserId { get; set; }
        public int SourceId { get; set; }
        public NotificationType NotificationType { get; set; }
        public bool IsChecked { get; set; }
        public DateTime CreateDate { get; set; }
    }
}

File: Diplom/Diplom/ClientApp/src/app/models/notifications/notification.ts
export enum NotificationType {
  AddedMusic = 1,
  AddedPlaylist = 2
}

export class Notification {
  notificationId: number;
  userId: number;
  sourceId: number;
  notificationType: NotificationType;
  message: string;
  createDate: Date;
  routeString: string;
}
